lms/djangoapps/verify_student/tests/test_views.py

```diff
@@ -1749,7 +1749,7 @@ def test_wrong_edx_id(self):
         mock.Mock(side_effect=mocked_has_valid_signature)
     )
     @patch('lms.djangoapps.verify_student.views.log.error')
-    @patch('lms.djangoapps.verify_student.utils.SailthruClient.send')
+    @patch('sailthru.sailthru_client.SailthruClient.send')
```

Comment:
Why this has been changed?

Reply:
As discussed, this was incorrectly patching the class in utils.py which was unused.